src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -1,34 +1,185 @@
+  private UserStore mockUserStore;
+  private TimeZone chicagoTimeZone = TimeZone.getTimeZone("America/Chicago");
+
+  private final UUID CONVERSATION_ID_ONE = UUID.randomUUID();
+  private final UUID USER_ID_ONE = UUID.randomUUID();
+  private final UUID USER_ID_TWO = UUID.randomUUID();
+  private final Message MESSAGE_ONE =
+      new Message(
+          UUID.randomUUID(),
+          CONVERSATION_ID_ONE,
+          USER_ID_ONE,
+          "message one",
+          Instant.ofEpochMilli(1000));
+          //Instant.parse("1969-12-31T18:00:01Z"));
+  private final Message MESSAGE_TWO =
+      new Message(
+          UUID.randomUUID(),
+          CONVERSATION_ID_ONE,
+          USER_ID_TWO,
+          "message two",
+          Instant.ofEpochMilli(4000));
+          //Instant.parse("1969-12-31T18:00:04Z"));
+  private final User USER_ONE =
+      new User(
+          USER_ID_ONE,
+          "test_username_one",
+          "$2a$10$/zf4WlT2Z6tB5sULB9Wec.QQdawmF0f1SbqBw5EeJg5uoVpKFFXAa",
+          Instant.ofEpochMilli(2000));
+          //Instant.parse("1969-12-31T18:00:02Z"));
+  private final User USER_TWO =
+      new User(
+          USER_ID_TWO,
+          "test_username_two",
+          "$2a$10$lgZSbmcYyyC7bETcMo/O1uUltWYDK3DW1lrEjCumOE1u8QPMlzNVy",
+          Instant.ofEpochMilli(5000));
+          //Instant.parse("1969-12-31T18:00:05Z"));
+  private final Conversation CONVERSATION_ONE =
+      new Conversation(
+          CONVERSATION_ID_ONE, USER_ID_ONE, "conversation_one", Instant.ofEpochMilli(3000));
+
+  @Before
+  public void setup() {
+    activityFeedServlet = new ActivityFeedServlet();
+    mockRequest = Mockito.mock(HttpServletRequest.class);
+    mockResponse = Mockito.mock(HttpServletResponse.class);
+    mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+    Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp"))
+        .thenReturn(mockRequestDispatcher);
+
+    mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+
+    mockConversationStore = ConversationStore.getTestInstance(mockPersistentStorageAgent);
+    activityFeedServlet.setConversationStore(mockConversationStore);
+
+    mockMessageStore = MessageStore.getTestInstance(mockPersistentStorageAgent);
+    activityFeedServlet.setMessageStore(mockMessageStore);
+
+    mockUserStore = UserStore.getTestInstance(mockPersistentStorageAgent);
+    activityFeedServlet.setUserStore(mockUserStore);
+
+    TimeZone.setDefault(chicagoTimeZone);
+  }
+
+  @Test
+  public void testDoGetNothing() throws IOException, ServletException {
+    List<Activity> noActivities = new ArrayList<>();
+
+    activityFeedServlet.doGet(mockRequest, mockResponse);
+
+    Mockito.verify(mockRequest).setAttribute("activities", noActivities);
+    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+  }
+
+  @Test
+  public void testDoGetOneOfEach() throws IOException, ServletException {
```

Comment:
Might be good to say why you aren't testing each activity type on its own (because you depend on the users, conversations, etc).

Or you should change these tests to use Mockito argument captors to check only the data you're interested in